cmd/ocm/create/cluster/cmd.go

```diff
@@ -377,6 +393,56 @@ func getVersionOptionsWithDefault(connection *sdk.Connection, channelGroup strin
 	return
 }
 
+func getSubscriptionTypeOptions(connection *sdk.Connection) ([]arguments.Option, error) {
+	options := []arguments.Option{}
+	billingModels, err := getBillingModels(connection)
+	if err != nil {
+		return options, err
+	}
+	for _, billingModel := range billingModels {
+		options = append(options, arguments.Option{
+			Value:       fmt.Sprintf("%s (%s)", billingModel.Id(), billingModel.Description()),
```

Comment:
Why do we need to add the description to the value?

Reply:
Added it before since interactive mode has a ? option to show description, but seems it only shows descriptions of the argument, not the drop down. Removed the description